packages/rocketchat-ui-account/client/accountIntegrations.js

```diff
@@ -0,0 +1,37 @@
+/* global */
+
+import toastr from 'toastr';
+
+Template.accountIntegrations.onCreated(function() {
+	this.subscribe('webdavAccounts');
```

Comment:
since [the other subscribe](https://github.com/RocketChat/Rocket.Chat/pull/11679/commits/8704199f5c6103a3027896c768ed636ca79b77da#diff-1fb17598b9dfd0ee77f53e204b70bec7R3) will always happen, there is no need to subscribe here.